### ISSUE TITLE
Add interface for parallel computing 

### ISSUE THREAD

**Issue by cvoegele** [i4Ds/Karabo-Pipeline#112]
Add interface for parallel computing with Daks, to allow for parallelizing trivial tasks.

- [x] Use DASK delayed
- [ ] Adapt the rest of karabo to be usable with it. (Make sure it waits, Use dask array instead of numpy array.)

**Comment by cvoegele** [comment]
@fschramka test on your machine --> https://github.com/i4Ds/Karabo-Pipeline/blob/add_new_metrics_sd/karabo/examples/plot_source_detection_exp.py

**Comment by deiruch** [comment]
1. We need more workflows to test this with
2. Make sure that our defaults work well for developer machines.
3. Make sure that performance is better
4. Make sure it works on CSCS

**Comment by deiruch** [comment]
#134 is currently blocked by this

**Comment by kenfus** [comment]
So, I timed the function `test_create_detection_from_ms` with vs. without dask and we "only" get a speed-up of around ~10% currently with dask in `imager.imaging_rascil`. I tried more/less cores/threads but the results were pretty consistent. `imaging_deconvolve` is a very long task and not parallelizable, thus many cores just wait around for another core to finish for it. 

Overall, the step in rascil is the one which takes the longest and it seems that dask currently does not really give a speed-up.
